.studio/sample-data-service

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#
+# Helper methods specific for the sample-data-service
+
+document "start_sample_data_service" <<DOC
+  Build and start the local sample-data-service
+DOC
+function start_sample_data_service() {
+  install_if_missing core/jq-static jq
+  install_if_missing core/grpcurl grpcurl
```

Comment:
Are these necessary here? You are not using either one.

Reply:
Yes, starting the service alone and installing `grpcurl` & `jq` would be handy so that `grpcurl & jq` can be used in hab studio.